package.json

```diff
@@ -6,19 +6,32 @@
   "bin": {
     "livestorm": "index.js"
```

Comment:
The "binary" will still work fine without having to install anything related to TS right ?

Reply:
Yes, there is no TypeScript in the `src` folder. The Types Definitions in `types` are resolved by JSDoc in the code editor. And finally the `.ts` files in the `__TESTS__` are transpiled by `@babel/preset-typescript` and is ofc not used by the CLI commands.